shared/src/common/types/types.ts

```diff
@@ -7,3 +7,4 @@ export {
 } from './master/master';
 export { type TenantResponseDto } from './tenant/tenant';
 export { type ValidationSchema } from './validation/validation';
+export { type TokenPayload } from './token/token-payload';
```

Comment:
```suggestion
export { type TokenPayload } from './token/token';
```
do the structure like in the other folders